Aerodynamics

RubiatMustak opened the pull request:
Please Accept My Request.

dwj300 committed:
repo

dwj300 committed:
Massive merge

dwj300 committed:
duplicate entry in .gitignore

dwj300 committed:
merging master

dwj300 committed:
Refactoring into utility method, adding abstractText everywhere, and fixing citationsController to use repository service

dwj300 committed:
Changing download from attachment to inline

dwj300 committed:
Refactoring indexUpdateManager

dwj300 committed:
Adding back in the safetext calls

dwj300 committed:
Fixing safetext

dwj300 committed:
gitignore

dwj300 committed:
Fixing filetypes method to match file system behavior

dwj300 committed:
Merge branch 'master' into repo

dwj300 committed:
Fixing mysql error

dwj300 committed:
Merge pull request #53 from SeerLabs/hotfix

Fixing mysql error

dwj300 committed:
Merge pull request #54 from SeerLabs/master

merging SQL hotfix into repo

dwj300 committed:
Commenting out hub urls

dwj300 committed:
connection pooling

fanchyna committed:
Merge branch 'master' of github.com:SeerLabs/CiteSeerX into slocation

fanchyna committed:
comment out /slocation links

fanchyna committed:
Merge pull request #58 from SeerLabs/slocation_removal

Commenting out hub urls

fanchyna committed:
Merge pull request #57 from SeerLabs/slocation

Slocation

fanchyna committed:
Update IncludeResultsSidebar.jsp

roll back to a change to this file. allenai search should be like others, rather than a "redirect".

dwj300 committed:
Using correct comment style

dwj300 committed:
Typo

fanchyna committed:
Merge pull request #56 from SeerLabs/connection_pool

connection pooling
tested by Douglas.

fanchyna committed:
Merge pull request #59 from SeerLabs/slocation

Slocation
tested on csxweb01 by Douglas and Jian.

fanchyna committed:
add csx_logo and csx_logo_front 7m-10m

dwj300 committed:
merging master changes

fanchyna committed:
csx_logo and csx_logo_front from 6m to 7m

fanchyna committed:
Merge pull request #60 from SeerLabs/csxlogo

csx_logo and csx_logo_front from 6m to 7m

dwj300 committed:
Merge branch 'master' into repo

dwj300 committed:
Merge pull request #41 from SeerLabs/repo

Repository API

dwj300 committed:
whoops merge conflict

dwj300 committed:
Fixing index updater typo

dwj300 committed:
Another typo

dwj300 committed:
Another typo2

dwj300 committed:
Refactoring FileSystemRepo

dwj300 committed:
Changing extensions to lower case

pychuang committed:
repository settings for index updater

pychuang committed:
Merge pull request #63 from pychuang/master

repository settings for index updater

sagnik committed:
changed url for semantic scholar

fanchyna committed:
Merge pull request #64 from sagnik/master

changed url for semantic scholar

dwj300 committed:
hotfix

dwj300 committed:
Reverting to original getFileTypes method

fanchyna committed:
Update web.xml

shorten session time out to 15 min

dwj300 committed:
defaulting to a FS setup.

fanchyna committed:
Merge pull request #65 from SeerLabs/pageviewlimit

add pageview limit filter

Commit:
change default repo to csxrepo01.ist.psu.edu

Commit:
Merge branch 'master' of github.com:SeerLabs/CiteSeerX

dwj300 committed:
adding connection close header

Commit:
Merge branch 'master' of github.com:SeerLabs/CiteSeerX

fanchyna committed:
allow searching non-ascii characters by encoding queries in UTF-8

fanchyna committed:
Merge pull request #66 from SeerLabs/latinguery

allow searching non-ascii characters by encoding queries in UTF-8

fanchyna committed:
replace solr and NSF logo's; update copyright ending date

fanchyna committed:
Merge pull request #67 from SeerLabs/solrlogo

replace solr and NSF logo's; update copyright ending date

Commit:
Merge branch 'master' of github.com:SeerLabs/CiteSeerX

for production

fanchyna committed:
Update gen_stat.py

md5 package is deprecated. replace it with hashlib.md5

fanchyna committed:
Update ranking.py

only get the first 10000 rankings for domains (full list contains 300,000+)

fanchyna committed:
close the last sitemap file, add logging and argparse

fanchyna committed:
fix bugs in generate_sitemap.py

fanchyna committed:
Update applicationContext-csx-jdbc.xml fix NullPointerError due to RedirectPDF

SEVERE: Servlet.service() for servlet [viewdoc] in context with path [] threw exception [Request processing failed; nested exception is java.lang.NullPointerException] with root cause
java.lang.NullPointerException
        at edu.psu.citeseerx.dao2.logic.CSXDAOImpl.getPDFRedirect(CSXDAOImpl.java:1758)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:601)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:304)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:182)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:149)
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:106)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:171)
        at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:89)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:171)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:204)
        at $Proxy10.getPDFRedirect(Unknown Source)
        at edu.psu.citeseerx.web.ViewDocController.handleRequest(ViewDocController.java:180)

Error messages in localhost.log files were detected for documents flagged as "5" (IS_PDFREDIRECT) in citeseerx.papers. This is due to accidentally deleted lines in applicationContext-csx-jdbc.xml after commit 36b758d by Douglas.

Commit:
Merge branch 'master' of github.com:SeerLabs/CiteSeerX

fanchyna committed:
remove pageview filter

the pageview filter (or governor)  may have some issues that prohibits Google from indexing our paper, although we do not have evidence for it yet. But for safety, we want to comment out the filter at this time.

fanchyna committed:
Create transfer_files.sh

Transfer extraction results from extraction server to local drive, then create XML files.

fanchyna committed:
Create ingest.sh

ingestion bash script. it must be run after transfer_files.sh

fanchyna committed:
Update applicationContext-updates.xml

changed a typo in indexupdateManager. FSrepo causes errors during ingestion. It should be named as "RESTrepo".

fanchyna committed:
Add repositoryService property to citeseerx object

pychuang committed:
repository_api/server.py: Douglas' production version

fanchyna committed:
Update IncludeFooter.jsp

remove NSF logo from front page as it is not longer a sponsor

fanchyna committed:
configuration file of sitemap generatioin script

fanchyna committed:
Create config.py

configuration file for Python scripts

Commit:
Merge branch 'master' of github.com:SeerLabs/CiteSeerX

Commit:
comment out homepage entry in author profile page

fanchyna committed:
upload citeseerx logos without the number

fanchyna committed:
remove 10M from logo

AaronNGray commented:
This is just too many different issues that should be in grouped or separate pull requests. Getting through and dealing with this lot is several days work. People accept small properly documented pull requests and tend to ignore ones like this as they are too much workload in one go !

Commit:
changed dates

fanchyna committed:
Create LICENSE.txt

shauryr committed:
Changed Line 15 - Commented close tag for c:if

fanchyna committed:
update .gitignore

fanchyna committed:
compatible with Python3

fanchyna committed:
compatible with Python3 and log files

fanchyna committed:
minor revision

fanchyna committed:
importing PDFs in WARC files

fanchyna committed:
write crawl metadata into CSV and import CSV to crawl DB and repo

bharathgit956 committed:
changes to handle grobid

changes to handle grobid and warn when the extractors in the xml file doesnt belong to"svm header parse", "grobid" or "parscit"

bharathgit956 committed:
enhancements related to grobid handling

fanchyna committed:
Merge pull request #79 from SeerLabs/CiteSeerX_grobid_handling

Cite seer x grobid handling
